src/main/java/ch/njol/skript/conditions/CondAnchorWorks.java

```diff
@@ -0,0 +1,58 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.World;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+public class CondAnchorWorks extends Condition {
+
+	static {
+		if (Skript.classExists("org.bukkit.block.data.type.RespawnAnchor"))
+			Skript.registerCondition(CondAnchorWorks.class, "respawn anchor[s] ([do]|1¦do(n't| not)) work[s] in %worlds%");
```

Comment:
```suggestion
			Skript.registerCondition(CondAnchorWorks.class, "respawn anchors [(do|1:do(n't| not))] work in %worlds%");
```
We're switching away from the broken pipe in favor of the colon

And just some other syntax suggestions